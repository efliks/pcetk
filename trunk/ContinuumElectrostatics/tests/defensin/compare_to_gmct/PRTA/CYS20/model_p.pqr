ATOM      0  C   THR    19      20.360  26.557   9.488   0.51000   1.700      0      0      0      0
ATOM      0  O   THR    19      19.824  26.699  10.639  -0.51000   1.500      0      0      0      0
ATOM      0  N   CYS    20      19.942  27.189   8.400  -0.47000   1.550      0      0      0      0
ATOM      0  H   CYS    20      20.378  27.057   7.508   0.31000   0.300      0      0      0      0
ATOM      0  CA  CYS    20      18.757  28.040   8.497   0.07000   1.700      0      0      0      0
ATOM      0  HA  CYS    20      18.344  28.035   9.498   0.09000   0.300      0      0      0      0
ATOM      0  CB  CYS    20      18.935  29.451   8.088   0.00000   1.700      0      0      0      0
ATOM      0  HB1 CYS    20      18.863  29.589   6.988   0.00000   1.000      0      0      0      0
ATOM      0  HB2 CYS    20      18.100  30.056   8.517   0.00000   1.000      0      0      0      0
ATOM      0  SG  CYS    20      20.538  30.135   8.673   0.00000   1.800      0      0      0      0
ATOM      0  HG1 CYS    20      21.327  29.313   7.993   0.00000   0.300      0      0      0      0
ATOM      0  C   CYS    20      17.725  27.435   7.575   0.51000   1.700      0      0      0      0
ATOM      0  O   CYS    20      18.116  26.780   6.619  -0.51000   1.500      0      0      0      0
ATOM      0  N   ILE    21      16.490  27.658   7.885  -0.47000   1.550      0      0      0      0
ATOM      0  H   ILE    21      16.223  28.206   8.674   0.31000   0.300      0      0      0      0
ATOM      0  CA  ILE    21      15.400  27.194   7.050   0.07000   1.700      0      0      0      0
ATOM      0  HA  ILE    21      15.748  26.680   6.165   0.09000   0.300      0      0      0      0
