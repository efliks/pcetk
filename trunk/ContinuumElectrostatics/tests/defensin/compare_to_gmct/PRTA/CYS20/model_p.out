Starting /home/mikolaj/local/bin/my_2diel_solver for molecule named scratch/PRTA/CYS20/site_p
using the following physical conditions:
Exterior dielectric constant, epsext = 80
Solvent probe radius, solrad = 1.4
Ion exclusion layer thickness, sterln = 2
Temperature, T = 300
Ionic strength, ionicstr = 0.1
Hueckel factor (epsext*kappasq) = 0.843038
Debye length 1/kappa = 9.7414
ln(10) * kBolt * T = 0.00413389
kBolt = 5.98442e-06
conconv = 0.000602214
econv = 332.063
Bohr radius = 0.529177
Proton Charge = 1
elementary_charge = 1.60218e-19
energy_unit = 2.30707e-18
Avogadro_Number = 6.02214e+23
voltconv = 14.3996
pkconv = 241.903
  Solute interior dielectric, epsin = 4
Vacuum dielectric constant, epsvac = 1

No blab level set (so no blabbing)
/home/mikolaj/local/bin/my_2diel_solver: WARNING:  The atom, "0:0:0:0:20:CB",
   occurs in both the solute and protein atom sets.
   For defining the Electrolyte boundary, the protein version
   will be used.
/home/mikolaj/local/bin/my_2diel_solver: WARNING:  The atom, "0:0:0:0:20:HB1",
   occurs in both the solute and protein atom sets.
   For defining the Electrolyte boundary, the protein version
   will be used.
/home/mikolaj/local/bin/my_2diel_solver: WARNING:  The atom, "0:0:0:0:20:HB2",
   occurs in both the solute and protein atom sets.
   For defining the Electrolyte boundary, the protein version
   will be used.
/home/mikolaj/local/bin/my_2diel_solver: WARNING:  The atom, "0:0:0:0:20:HG1",
   occurs in both the solute and protein atom sets.
   For defining the Electrolyte boundary, the protein version
   will be used.
/home/mikolaj/local/bin/my_2diel_solver: WARNING:  The atom, "0:0:0:0:20:SG",
   occurs in both the solute and protein atom sets.
   For defining the Electrolyte boundary, the protein version
   will be used.
Using finite difference method with lattice levels:
                   Centering Style   grid dimension    spacing
Coarsest level: ON_GEOM_CENT resolved to (18.5335, 28.0044, 8.13276)) 121 2
Finer level:    (19.5525, 29.7088, 8.05171) 101 1
Finer level:    (19.5525, 29.7088, 8.05171) 101 0.5
Finest level:   (19.5525, 29.7088, 8.05171) 101 0.25


Self energy of scratch/PRTA/CYS20/site_p in two dielectric environment = 23.1412
(probably in kcal/mole, INCLUDING GRID ARTEFACTS)


Interaction energy of scratch/PRTA/CYS20/site_p with background charge set in two dielectric environment = -0.0588875
(probably in kcal/mole)
Total runtime: 8.85 s user 0.06 s system 
